src/test/kotlin/io/nais/security/oauth2/registration/ClientStoreTest.kt

```diff
@@ -48,5 +54,68 @@ internal class ClientStoreTest {
         }
     }
 
+    @Test
+    fun findClients() {
+        withMigratedDb {
+            with(ClientStore(DataSource.instance)) {
+                val testclient = oauth2Client("testclient")
+                val testclient2 = oauth2Client("testclient2")
+                storeClient(testclient)
+                storeClient(testclient2)
+                findClients(listOf("testclient", "testclient2")) shouldBe mapOf(
+                    "testclient" to testclient,
+                    "testclient2" to testclient2
+                )
+            }
+        }
+    }
+
+    @Test
+    fun testFindClientsAndStorePerformance() {
```

Comment:
These tests might be flaky depending on the specific machine that the tests run on. I think it's fine for now, though we might want to re-assess if flakiness becomes an issue